src/main/java/io/vertx/core/json/JsonObject.java

```diff
@@ -105,7 +105,7 @@ public static JsonObject of() {
    * @return a JsonObject containing the specified mapping.
    */
   public static JsonObject of(String k1, Object v1) {
-    JsonObject obj = new JsonObject(new LinkedHashMap<>(1));
+    JsonObject obj = new JsonObject(new LinkedHashMap<>(3));
```

Comment:
```suggestion
    JsonObject obj = new JsonObject(new LinkedHashMap<>(2));
```

According to https://guava.dev/releases/21.0/api/docs/src-html/com/google/common/collect/Maps.html#line.291